src/test/lib/utils/pruning_utils_test.cpp

```diff
@@ -0,0 +1,80 @@
+#include "base_test.hpp"
+
+#include "utils/column_pruning_utils.hpp"
+
+namespace hyrise {
+
+class PruningUtilsTest : public BaseTest {};
+
+TEST_F(PruningUtilsTest, ColumnIDMapping) {
```

Comment:
`GTEST_SKIP()` to make clear we do not run an (empty) test in release builds?

Reply:
Just noticed that this test is `pruning_utils_test.cpp` where the utils tested are in `column_pruning_utils.hpp`. Could we make this consistent?

Reply:
That's the last remaining todo. I wanted to do it after acceptance as Github as the changes are often displayed as completely new files (since I mostly screw up the renaming ... ).